### ISSUE TITLE
Can't install bindfs on Debian guest

### ISSUE THREAD

**Issue by xcambar** [gael-ian/vagrant-bindfs#72]
Hi,
I'm trying to get bindfs from a OSX host to a debian (testing) guest.

I get this error while provisioning:

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

apt-get update

Stdout from the command:

Get:1 http://security.debian.org/debian-security stretch/updates InRelease [62.9 kB]
Get:2 http://security.debian.org/debian-security stretch/updates/main Sources [712 B]
Get:3 http://cdn-fastly.deb.debian.org/debian stretch InRelease [186 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian stretch/main Sources.diff/Index [27.9 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian stretch/main Sources.diff/Index [27.9 kB]
Get:5 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 Packages.diff/Index [27.9 kB]
Get:5 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 Packages.diff/Index [27.9 kB]
Get:6 http://cdn-fastly.deb.debian.org/debian stretch/main Sources [6,780 kB]
Get:7 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 Packages [7,108 kB]
Reading package lists...


Stderr from the command:

mesg: ttyname failed: Inappropriate ioctl for device
E: Could not open file /var/lib/apt/lists/httpredir.debian.org_debian_dists_stretch_main_source_Sources.diff_Index - open (2: No such file or directory)
E: Could not open file /var/lib/apt/lists/httpredir.debian.org_debian_dists_stretch_main_binary-amd64_Packages.diff_Index - open (2: No such file or directory)
```

Here's the line in the Vagrantfile:

```
  config.bindfs.bind_folder "/path/to/dir", "/var/endpoint"
```

It looks like a permission issue... Do you have any advice to get it up and running?

**Comment by gael-ian** [comment]
Hi @xcambar,

You're right, it looks like a permission issue. Without more information's, I'd say your base box was not build to give administrative power to the `vagrant` user or you're logging into your box with a different user.

Can you post your full Vagrantfile and any other configuration file that may be usefull? And can you try with another base box?

Thanks

**Comment by NicolasGoerlach** [comment]
Same Issue here with ARTACK/debian-jessie box.

```
config.vm.box = "ARTACK/debian-jessie"
config.vm.synced_folder "./www", "/var/www/html", id: "application", type: 'nfs'
config.bindfs.bind_folder "/var/www/html", "/www", perms: "u=rw:g=r:o=r", create_as_user: true
```

Error Message:

> ==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Configuring and enabling network interfaces...
==> default: Exporting NFS shared folders...
==> default: Preparing to edit /etc/exports. Administrator privileges will be required...
==> default: Mounting NFS shared folders...
==> default: Mounting shared folders...
    default: /vagrant => /Users/nicolas/dev/cto4
==> default: Checking for bindfs in VM...
    default: Package manager detected: apt-get
==> default: Fuse seems to not be installed on the virtual machine, installing now
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!
apt-get install -y fuse
Stdout from the command:
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  fuse
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 70.4 kB of archives.
After this operation, 112 kB of additional disk space will be used.
Err http://ftp.ch.debian.org/debian/ jessie/main fuse amd64 2.9.3-15+deb8u1
  404  Not Found [IP: 129.132.53.171 80]
Err http://security.debian.org/ jessie/updates/main fuse amd64 2.9.3-15+deb8u1
  404  Not Found [IP: 195.20.242.89 80]
Stderr from the command:
E: Failed to fetch http://security.debian.org/pool/updates/main/f/fuse/fuse_2.9.3-15+deb8u1_amd64.deb  404  Not Found [IP: 195.20.242.89 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?

**Comment by gael-ian** [comment]
Hi @NicolasGoerlach 

Your problem is slightly different. It's not an `apt-get upgrade` operation that fails but some package installation. Checking the exact code for Debian guest, I realized it missed an update of package repositories before trying to install fuse.

As the base box you use is pretty old (2 years!), the list of available packages is not reliable anymore and have to be updated prior to any installation.

I just released vagrant-bindfs 1.0.8 that should fix this.

Thanks.
